### ISSUE TITLE
A question about the evaluation time

### ISSUE THREAD

**Issue by JaehongJaehongMin** [LQTS/VTDexManip#2]
Hello,
Thank you for your great work!
The code is highly reproducible, and I was able to run the training and evaluation without issues.

I have a question regarding the time it takes to evaluate the trained checkpoints.

As you mentioned in the paper, training the pretrained model with RL took about a day.
Now, I'm evaluating the trained checkpoints, but I'm not sure how long the evaluation will take.
Could you please let me know how long it usually takes?

**Comment by LQTS** [comment]
> Hello, Thank you for your great work! The code is highly reproducible, and I was able to run the training and evaluation without issues.
> 
> I have a question regarding the time it takes to evaluate the trained checkpoints.
> 
> As you mentioned in the paper, training the pretrained model with RL took about a day. Now, I'm evaluating the trained checkpoints, but I'm not sure how long the evaluation will take. Could you please let me know how long it usually takes?

Hi, thank you very much for your kind words and for taking the time to reproduce our results!

Regarding your question: the "one day" mentioned in the paper refers specifically to the time spent on pretraining the model, not the combined time of pretraining and reinforcement learning (RL). The training times for these two stages are discussed separately in the supplementary material:

-     Pretraining details are provided in Section B.2.1

-     Reinforcement learning details are in Section B.4

During the RL phase, running 1000 iterations typically takes about 7 hours with an Intel Xeon Gold 6326 processor and an NVIDIA 3090 GPU. The total evaluation time may vary depending on hardware and evaluation settings, but hopefully this gives you a useful reference point.

Let us know if you have any further questions!

**Comment by JaehongJaehongMin** [comment]
Sorry for the confusing expression. I meant the RL phase when I said 'training.

I ran the evaluation code, and it took more than five days.
Since the training code finished in less than a day, I think the issue is with the evaluation code.

**Comment by LQTS** [comment]
> Sorry for the confusing expression. I meant the RL phase when I said 'training.
> 
> I ran the evaluation code, and it took more than five days. Since the training code finished in less than a day, I think the issue is with the evaluation code.

Thank you very much for your feedback. We have updated evaluation.py and modified the evaluation command as follows:
`python eval_agent.py --task bottle_cap-vt_all_cls --rl_device cuda:0 --resume_model runs/BottleCap/bottle_cap/bottle_cap-vt_all_cls/seed111/checkpoint/model_2000.pt --test --seed 111
`
With this update, evaluating the trained models now takes less than one hour.

**Comment by JaehongJaehongMin** [comment]
I ran the evaluation code last night, and it reproduced well, showing results almost identical to those in your paper.
Thank you again! I really appreciate it!

**Comment by hdjlcbz** [comment]
Hello!
See your question
I have been trying to reproduce the success rates mentioned in your ICLR 2025 paper "VTDexManip: A DATASET AND BENCHMARK FOR VISUAL-TACTILE PRETRAINING AND DEXTEROUS MANIPULATION WITH REINFORCEMENT LEARNING" by running the train_agent.py to train the RL policy and then using eval_agent to measure the success rate. However, I found that the success rates I obtained are quite different from those reported in the paper.
I'm wondering if there are some specific configurations or settings that I might have missed, which could lead to such discrepancies. Could you please provide some guidance on the key configurations or additional steps that should be taken to achieve success rates similar to those in the paper?
